aten/src/ATen/Context.cpp

```diff
@@ -121,6 +121,14 @@ void Context::setAllowTF32CuDNN(bool b) {
   allow_tf32_cudnn = b;
 }
 
+bool Context::allowTF32Mkldnn() const {
```

Comment:
@yanbing-j , should we use "onednn" rather than "mkldnn"? Because, I noticed you maintain a huge ghstack to replace mkldnn with onednn.

Reply:
@EikanWang Is this a new API which conains 'mkldnn'? If so, you should use 'onednn' directly. Otherwise, I will change it to 'onednn' in my ghstack when this PR is merged.
For this new added API, you can use 'onednn' directly. For other existing API usage, you can use 'mkldnn', and I will rebase for that.

Reply:
Yes, this is a new API. @ZhiweiYan-96 let's refine the API name by replacing `mkldnn` with `onednn`. Meanwhile, pls. link @yanbing-j's PR to help reviewers get the background.

Reply:
ghstack starts from https://github.com/pytorch/pytorch/pull/133289.

Reply:
sure, I will change the word, and thanks for your kindly clarification.

Reply:
I have changed word "mkldnn" need to be changed in the PR, thanks for your comments.